pkgs/by-name/nu/nuget-to-json/nuget-to-json.sh

```diff
@@ -0,0 +1,119 @@
+#!@runtimeShell@
```

Comment:
This is presumably almost all duplicate code, so I think it would be better to just add have one package that can output both formats.

Reply:
Yes, I guess it would. I'll look into it.

Reply:
Actually, I've been thinking about this, and I think I'd rather leave `nuget-to-nix` as it is and add new features into `nuget-to-json`.

That should be another incentive for people to migrate to the JSON based ones, maintaining both the nix and JSON output wouldn't be a good use of resources considering the long term objective is to get rid of the nix based ones anyways.

Reply:
I still don't think that's a good argument for duplicating code.  We can always add backwards compatibility to the new implementation and strip it out later, or if it becomes a pain to maintain/test.

Forking it like this just means one will rot.

Reply:
Deduplicated by making it just convert the output of `nuget-to-json` to nix.